components/image-builder-mk3/debug.sh

```diff
@@ -5,8 +5,12 @@
 docker ps &> /dev/null || (echo "You need a working Docker daemon. Maybe set DOCKER_HOST?"; exit 1)
 gcloud auth list | grep typefox &>/dev/null || (echo "Login using 'gcloud auth login' for the docker push to work"; exit 1)
 
-leeway build .:docker -Dversion=dev
-devImage=eu.gcr.io/gitpod-dev/image-builder:dev
+leeway build -v .:docker -Dversion=cw-dev -DimageRepoBase=eu.gcr.io/gitpod-core-dev/build
+devImage=eu.gcr.io/gitpod-core-dev/build/image-builder-mk3:cw-dev
 
-kubectl patch deployment image-builder --patch '{"spec": {"template": {"spec": {"containers": [{"name": "service","image": "'$devImage'"}]}}}}'
-kubectl get pods --no-headers -o=custom-columns=:metadata.name | grep image-builder | xargs kubectl delete pod
\ No newline at end of file
+kubectl patch deployment image-builder-mk3 --patch '{"spec": {"template": {"spec": {"containers": [{"name": "image-builder-mk3","imagePullPolicy":"Always","image": "'$devImage'"}]}}}}'
+kubectl rollout restart deployment/image-builder-mk3
+kubectl rollout status -w deployment/image-builder-mk3
+# give the old pod time to disappear
+sleep 20
+gpctl debug logs image-builder-mk3
```

Comment:
```suggestion
gpctl debug logs image-builder-mk3

```

Reply:
I'm not sure I understand the comment :)

Reply:
This is the fix that the github warns about because there is no new line. There is no specific problem, so it is a minor nits.
https://thoughtbot.com/blog/no-newline-at-end-of-file